select_statements.sql

```diff
@@ -20,10 +20,45 @@ WHERE vehicle.stationName = "Uppsala station" AND booking.endDatum < "2024-04-13
 
 -- Underhållspersonal
 -- Sök fram alla bilar i behov av kontroll
+SELECT vehicle.registrationNumber FROM green_rental.vehicle
+LEFT JOIN booking_details ON vehicle.registrationNumber = booking_details.registrationNumber
+LEFT JOIN booking ON booking_details.bookingNumber = booking.bookingNumber
+LEFT JOIN control ON vehicle.registrationNumber=control.registrationNumber
+WHERE control.controlLarge is FALSE and control.datum < "2024-04-12" and booking.endDate = "2024-04-12";
+
 -- Sök fram alla bilar i behov av kontroll inom 3/6/12 månader
+SELECT vehicle.registrationNumber
+FROM green_rental.vehicle
+LEFT JOIN control ON control.registrationNumber=vehicle.registrationNumber
+WHERE control.controlLarge is TRUE and control.datum>=DATE_SUB(CURDATE(), INTERVAL 3 MONTH);
```

Comment:
Should also have control.datum is NULL.
Should probably be less than date + 3 month.

Could perhaps use https://www.w3schools.com/mysql/func_mysql_date_add.asp